docs/01-startgids.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Startgids
+---
+# Introductie
+
+De DSO- toolkit is bedoeld om een set van ontwerpprincipes, componenten en templates aan te bieden. Waarmee een consistente gebruikerservaring gerealiseerd mee kan worden. Het helpt ook tijd en middelen te besparen bij het door ontwikkelen of starten van een platform met betrekking tot het omgevingsloket. 
+
+## Disclaimer
+De Dynamische Stijlgids is bedoeld voor gebruik binnen het programma Aan de Slag met de Omgevingswet. Het is daarom niet toegestaan dat derden - zonder voorafgaande schriftelijke toestemming van het programma Aan de Slag met de Omgevingswet - de informatie en/of de huisstijl kopiëren, vermenigvuldigen, bewerken of doorleveren anders dan voor persoonlijk, niet-commercieel gebruik.
+
+## Een platform, proces of dienst is nooit af, daarom iteratief ontwerpen.
+Iteratief ontwerpen is een continu verbeteringsproces van een website, product of dienst. Dit wordt georganiseerd door verschillende cycli van tests, gebruikersevaluaties  te organiseren. Om daar verbeteringen te vinden en deze in het product of dienst te verwerken om deze vervolgens weer te testen.  
+De beste manier om deze tests uit te voeren is in zo’n realistisch mogelijke condities. Wees daarom voorbereid op het moment van live gaan met een testplan om meteen inzichten te verzamelen zodat de cyclus van verbeteren en testen meteen kan beginnen.  
+
+Dit betekent dat het platform waar je aan gaat werken nooit compleet af zal zijn. De developers, ontwerpers en stakeholders zijn altijd op zoek naar manieren om de gebruikerservaring te verbeteren.  
+Zijn er in de toekomst verbeteringen die zo essentieel of belangrijk zijn dat ze opgenomen moeten worden in de toolkit. Kijk dan naar de procespagina over hoe je kan bijdragen aan de toolkit.
+
+<a>Gebruikerstesten organiseren en resultaten verwerken(onder ontwikkeling)</a></br>
```

Comment:
Als je hier Markdown gebruikt, is de `<br>` niet nodig.. Overigens is het niet `</br>`. BR is een selfclosing tag. HTML5 geeft aan dat `<br>` genoeg is. Als je graag expliciet wil aangeven dat het een selfclosing tag is kun je voor `<br />` of `<br/>` gaan.

Maar: Ga voor Markdown, dan hoef je hier niet over na te denken.